README.md

```diff
@@ -215,7 +215,7 @@ app.use exprestive
   * default: `__dirname`
 * `controllersPattern`
   * [Glob](https://github.com/isaacs/node-glob) pattern used to find controllers. Resolved relative to `appDir`
-  * default: `'controllers/*_controller.{coffee,js}'`
+  * default: `'controllers/*_controller.{coffee,js,ls}'`
```

Comment:
Please remove the underscore and the `c`, so that it reads:

```
default: `'controllers/*ontroller.{coffee,js,ls}'
```

People might want to call their controllers `foo-controller.ls` or `FooController.ls`, and we want to support that as well.